internal/testhelpers/e2e/session.go

```diff
@@ -614,45 +568,36 @@ func (s *Session) SvcLog() string {
 	return fmt.Sprintf("Could not find state-svc log, checked under %s, found: \n%v\n, files: \n%v\n", logDir, lines, files)
 }
 
-func (s *Session) MostRecentStateLog() string {
-	rx := regexp.MustCompile(`state-\d`)
+func (s *Session) DebugLogs() string {
 	logDir := filepath.Join(s.Dirs.Config, "logs")
 	if !fileutils.DirExists(logDir) {
-		return ""
+		return "No logs found in " + logDir
 	}
-	var result string
-	var newest time.Time
+
+	var sectionStart, sectionEnd string
+	sectionStart = "\n=== "
+	if os.Getenv("GITHUB_ACTIONS") == "true" {
+		sectionStart = "##[group]"
+		sectionEnd = "##[endgroup]"
+	}
+
+	result := "Logs:\n"
```

Comment:
nit: For some reason, this line appears to be indented in the test failures. It's not immediately clear to me why so this might be more trouble to "fix" than it's worth.